internal/app/backend/backend_service_test.go

```diff
@@ -32,6 +37,203 @@ import (
 	statestoreTesting "open-match.dev/open-match/internal/statestore/testing"
 )
 
+// This unit test only covers the failure cases as a mmf service is required to get response successfully.
+func TestDoFetchMatchesInChannel(t *testing.T) {
+	assert := assert.New(t)
+
+	insecureCfg := viper.New()
+	secureCfg := viper.New()
+	secureCfg.Set("tls.enabled", true)
+	restFuncCfg := &pb.FetchMatchesRequest{
+		Config:  &pb.FunctionConfig{Name: "test", Type: &pb.FunctionConfig_Rest{Rest: &pb.RestFunctionConfig{Host: "om-test", Port: int32(666)}}},
+		Profile: []*pb.MatchProfile{{Name: "1"}, {Name: "2"}},
+	}
+	grpcFuncCfg := &pb.FetchMatchesRequest{
+		Config:  &pb.FunctionConfig{Name: "test", Type: &pb.FunctionConfig_Grpc{Grpc: &pb.GrpcFunctionConfig{Host: "om-test", Port: int32(666)}}},
+		Profile: []*pb.MatchProfile{{Name: "1"}, {Name: "2"}},
+	}
+
+	tests := []struct {
```

Comment:
For table tests it needs to be clear what the test case is. There's typically a generated method name to embed the description so devs can understand what the scenario is trying to do. Also generally you don't want to do testify asserts in table tests for this reason.

Please update the following tests. For this file it seems it's best to add a description field and print that out in the .Run function.

See: https://github.com/googleforgames/open-match/blob/master/tools/certgen/internal/internal_test.go#L98 for an example.